Baby Monitor/Source Files/Services/Connection/WebRTC/WebRtcServerManagerProtocol.swift

```diff
@@ -13,6 +13,12 @@ protocol WebRtcServerManagerProtocol {
     /// Closes connection
     func stop()
 
+    /// Pauses video streaming.
+    func pauseMediaStream()
```

Comment:
This is why I usually don't write method descriptions in a protocols - 9/10 cases, method names are self explanatory ;) 
I do understand that this is a project convention, but it looks funny nonetheless ;)

Reply:
In this case I agree, but actually there is no convention of adding documentation in the project overall. But as long it was documented here, I decided to add to these two methods too :)